README.md

```diff
@@ -17,6 +18,24 @@ This library provides the ability to process messages passed onto a dataLayer qu
 - [License](#license)
   
 
+## Quick Start
+First, install the package with your favorite package manager
+```bash
+npm install data-layer-helper --save-dev
+// or
+yarn add data-layer-helper --dev
```

Comment:
I would not make the assumption that DLH is a dev dependency - libraries like Measurement Library will need it as a regular dependency. Other tools may need it as a regular dependency too if they are taking advantage of things like the command API or listener.